covid19-dashboard/cron.yaml

```diff
@@ -0,0 +1,5 @@
+cron:
+   - description: 'keep data fresh'
+     url: /get-all-data
+     schedule: every 5 minutes
```

Comment:
does this cron still work? I would suggest to make this every hour

Reply:
cron works, i updated to every hour.

Reply:
Here still says "every 5 minutes"?

Reply:
Fixed.